third_party/rust-hypervisor-firmware-boot/src/paging.rs

```diff
@@ -38,9 +38,15 @@ pub fn setup() {
     for (i, l2) in l2s.iter().enumerate() {
         l3[i].set_addr(phys_addr(l2), pt_flags);
     }
+    // Upper half hack.
```

Comment:
Perhaps add a bit more a description for why this works for future reference.

Reply:
Done.